### PR TITLE
Shrink size a little

### PR DESCRIPTION
**Pull request by KyleMaas**
This gets rid of a few unused dependencies.  Doesn't seem to actually do much to the output file size in debug builds, but probably good for housekeeping.

Takes the release build from 4520557 bytes to 4518019 bytes.

See #219.

### CODE REVIEW

**Comment by KyleMaas** [comment]
Added terser to minify the output.  Release build size is now 4369664 bytes.  Still seems to work just fine.

* Tried terser with mangling.  Shrinks the size significantly, but does not work.
* Tried terser with only one compress pass.  Two passes does actually make the output smaller.

File: package.json [hunk 1]
```diff
@@ -24,7 +22,6 @@
     "ssb-markdown": "^6.0.7",
     "ssb-mentions": "^0.5.2",
     "ssb-ref": "^2.13.3",
-    "ssb-sort": "^1.1.3",
```
Comment (arj03) [inline package.json:27]: this is used in ui/threads.js

**Comment by arj03** [comment]
It's very nice to remove these unused stuff.

File: package.json (hunk 1 above)
Comment (KyleMaas) [reply package.json]: Ah, you're right.

**Comment by arj03** [comment]
Nice thanks :)

**Comment by KyleMaas** [comment]
Unfortunately, it's not much.  But it's also not nothing.

**Pull request merged**